src/neo/Cryptography/ECC/ECPoint.cs

```diff
@@ -128,23 +133,30 @@ public static ECPoint DeserializeFrom(BinaryReader reader, ECCurve curve)
             }
         }
 
-        public byte[] EncodePoint(bool commpressed)
+        public byte[] EncodePoint(bool commpressed, bool allowCache = true)
```

Comment:
What is the purpose of allowCache?

Reply:
I want to avoid the modification of this array inside the VM, but I changed the logic